src/migrations/20220819014844-user-settings-table.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => {
+    await queryInterface.createTable(
+      'UserSettings',
+      {
+        id: {
+          type: Sequelize.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+          allowNull: false,
+        },
+        // 'email', etc.
+        class: { type: Sequelize.STRING, allowNull: false },
+        key: { type: Sequelize.STRING, allowNull: false },
+        default: { type: Sequelize.STRING, allowNull: false },
```

Comment:
can we change this to a jsonb datatype, this would allow complex settings to be saved, Like saving filters

Reply:
that's a great idea - changed here: 5fbf645